components/input/ColorPicker.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-components';
+import { ChromePicker } from 'react-color';
+
+const ColorPicker = ({ text, initialColor, onColorChange, ...rest }) => {
+    const [pickerState, setPickerState] = useState({
+        display: false,
+        color: initialColor
+    });
+
+    const rgbaColor = (color) => {
+        return `rgba(${color.rgb.r}, ${color.rgb.g}, ${color.rgb.b}, ${color.rgb.a})`;
+    };
+
+    const handleClick = () => {
+        setPickerState((prevState) => {
+            return {
+                ...prevState,
+                display: !pickerState.display
+            };
+        });
+    };
+
+    const handleClose = () => {
+        setPickerState((prevState) => {
+            return {
+                ...prevState,
+                display: false
+            };
+        });
+    };
+
+    const handleChange = (color) => {
+        setPickerState((prevState) => {
+            return {
+                ...prevState,
+                color: color
+            };
+        });
+    };
+
+    const popover = {
```

Comment:
Can you define CSS in a separate file and import it please? Add a `className` to the component if needed.

Reply:
I moved the definitions to some auxiliary CSS, but I don't know what are the good practices for CSS or where to put this auxiliary file. Probably some changes to what I did are required.

As designed at present, if a `className` is specified, it would go under `...rest` and passed down to the button. Do you think a different functionality is needed?

Reply:
As you did is OK for me. Just rename the file to `ColorPicker.css` to be consistent with the component name ;)